### PR TITLE
Feature/13817 change color scheme

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
Implement green colour scheme with gradient.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13817

## Screenshots
Screen Shot 2022-09-02 at 16 48 46

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 1]
```diff
@@ -198,6 +203,15 @@ class GradientView: UIView {
 			gradientLayer.startPoint = CGPoint(x: 0.0, y: 0.0)
 			gradientLayer.endPoint = CGPoint(x: 1.0, y: 1.0)
 			
+		case .solidGreen:
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift:206]: It's a gradient, I would rather rename the other greens to solidLightGreen and solidDarkGreen

```suggestion
		case .green:
```

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift [hunk 2]
```diff
@@ -389,4 +389,23 @@ final class HealthCertificateViewModel {
 		isPrimaryFooterButtonEnabled = healthCertificate.validityState != .blocked && healthCertificate.validityState != .revoked
 	}
 
+	private var isCertificatMostRelevant: Bool {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift:392]:
```suggestion
	private var isMostRelevantCertificate: Bool {
```

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift (hunk 1 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Views/GradientView.swift]: 👍 done

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift (hunk 2 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift]: 👍  done

**Review by flxschmidt — APPROVED** [review]

**Pull request merged**